fastlane_core/spec/tool_collector_spec.rb

```diff
@@ -1,123 +0,0 @@
-describe FastlaneCore::ToolCollector do
-  before(:all) { ENV.delete("FASTLANE_OPT_OUT_USAGE") }
-
-  let(:collector) { FastlaneCore::ToolCollector.new }
-
-  it "keeps track of what tools get invoked" do
-    collector.did_launch_action(:scan)
-
-    expect(collector.launches[:scan]).to eq(1)
-    expect(collector.launches[:gym]).to eq(0)
-  end
-
-  it "tracks which tool raises an error" do
-    collector.did_raise_error(:scan)
-
-    expect(collector.error).to eq(:scan)
-    expect(collector.crash).to be(false)
-  end
-
-  it "tracks which tool crashes" do
-    collector.did_crash(:scan)
-
-    expect(collector.error).to eq(:scan)
-    expect(collector.crash).to be(true)
-  end
-
-  it "does not post the collected data if the opt-out ENV var is set" do
-    with_env_values('FASTLANE_OPT_OUT_USAGE' => '1') do
```

Comment:
Do we have an equivalent test, that makes sure we're not sending any data to any backend when `FASTLANE_OPT_OUT_USAGE` is set?

Reply:
Good question